src/game.cpp

```diff
@@ -2170,6 +2168,59 @@ bool game::handle_mouseview( input_context &ctxt, std::string &action )
     return true;
 }
 
+tripoint mouse_edge_scrolling( input_context ctxt __attribute__( ( unused ) ), const int speed )
```

Comment:
You can `(void) ctxt;` in function body if you don't want to use this parameter.

```
ctxt __attribute__( ( unused ) )
```
is something which is probably not supported by all C++ compilers.

Reply:
Cheers, fixed now!